cstool/cstool.c

```diff
@@ -354,9 +354,9 @@ static void usage(char *prog)
 static void print_details(csh handle, cs_arch arch, cs_mode md, cs_insn *ins)
 {
 	printf("\tID: %u (%s)\n", ins->id, cs_insn_name(handle, ins->id));
-	if (ins->is_alias) {
+	if (ins->flags & CS_INSN_FLAG_ALIAS) {
```

Comment:
I think a function to check for flags is better for the `flag` field (something like `cs_flag_set(insn->flags, CS_INSN_FLAG_ALIAS)`). Bit operations in conditions get messy after a while.

Reply:
I've added some macros to manage flags for more complex cases. See `capstone.h`.

```
CS_INSN_FLAGS_ALL -- test if all flags is set
CS_INSN_FLAGS_ANY -- test if any flag is set
CS_INSN_FLAGS_SET -- enable flags if condition is true, disable flags if false
```

But forget to add some documentation. :)

Reply:
Ouh, sorry. Overlooked it. Just took a shallow look.

Reply:
Wat. I meant for checking if a flag is set. Like `cs_flag_is_set(insn->flags, CS_INSN_FLAG_ALIAS)` => `true/false`.
But I'll review tomorrow again with a clear mind.